### PR TITLE
Area pages 89930432

### PR DESCRIPTION
**Pull request by variousred**
https://www.pivotaltracker.com/story/show/89930432

### CODE REVIEW

**Comment by variousred** [comment]
@levibrown @thescubageek @dreuss

File: public/area_page.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 function app() {
```
Comment (dreuss) [inline public/area_page.js]: Any adversity to name-spacing this function to something like area_page_app() ?  app() seems really general, but maybe there's a reason for that naming, maybe this file is compiled from somewhere else & the name is auto-generated.  A name-spaced function would help any future developers with a little clue as to what/where the function does/exists.

Comment (variousred) [inline public/area_page.js]: @dreuss good point. And you're right, it does look like this was compile from coffee, but who knows where the source file is. @bbauer ? Either way, I'll change this to what you suggest Dave.

Comment (bbauer) [inline public/area_page.js]: Good question. Looking...

**Pull request merged**